libs/recnet-api-model/src/lib/model.ts

```diff
@@ -57,3 +57,15 @@ export const inviteCodeSchema = z.object({
   usedAt: dateSchema.nullable(),
 });
 export type InviteCode = z.infer<typeof inviteCodeSchema>;
+
+export const announcementSchema = z.object({
+  id: z.number(),
+  title: z.string(),
+  content: z.string(),
+  startAt: dateSchema.transform((v) => new Date(v)),
+  endAt: dateSchema.transform((v) => new Date(v)),
```

Comment:
```suggestion
  startAt: dateSchema,
  endAt: dateSchema,
```

I remembered we had a discussion before and our consensus was used ISO Date string as Date in DTO. Therefore, `dateSchema` would be the right schema?

Reply:
fixed